django/core/cache/backends/db.py

```diff
@@ -202,15 +209,35 @@ def _base_set(self, mode, key, value, timeout=DEFAULT_TIMEOUT):
                 return True
 
     def delete(self, key, version=None):
-        key = self.make_key(key, version=version)
-        self.validate_key(key)
+        self.delete_many([key], version)
```

Comment:
similar to `get()` above, I appreciate the code reuse, but I personally would prefer if `get()` and `delete()` were not touched.

Reply:
I think it's fine to tidy up here, we're basically just inverting the reuse where previously `delete_many` reused `delete`